x-pack/filebeat/module/squid/log/test/access1.log-expected.json

```diff
@@ -6059,4 +6059,4 @@
         "url.top_level_domain": "com",
         "user.name": "badeyek"
     }
-]
\ No newline at end of file
+]
```

Comment:
Is this whitespace change expected?

Reply:
Not necessary, my editor must have added that automatically on save. Will remove.

Reply:
This has led me down a rabbit whole because both vim and VSCode are adding a trailing newline on save and won't let me remove it. I am just going to leave this here as long as the test passes rather than trying to find which editor configuration lets me remove it.

Reply:
There's an entry in [.editorconfig](https://github.com/elastic/beats/blob/2a26f9af28835415d28ba8b439666ce1b9b12bc1/.editorconfig#L8) that enforces trailing newlines. This is almost always correct, but not with expected test outcome JSON files. I turn off editorconfig awareness in the editor I use because of this.